src/odb/test/cpp/sky130_test_fixture.h

```diff
@@ -26,7 +26,9 @@ class Sky130TestFixutre : public ::testing::Test
         db_.get(), &logger_, /*ignore_non_routing_layers=*/false);
     lib_ = OdbUniquePtr<odb::dbLib>(
         lef_reader.createTechAndLib(
-            "sky130", "sky130", "data/sky130hd/sky130_fd_sc_hd.tlef"),
+            "sky130",
+            "sky130",
+            "src/odb/test/data/sky130hd/sky130_fd_sc_hd.tlef"),
```

Comment:
This is going to break the ctest version of the test. We should probably introduce some helper function that checks if the env is bazel or ctest and modify the path.

Reply:
yeah, I suspect so. Since I can't run the cmake build (that is like soo last week), I'll let the CI figure out what is happening :)

Reply:
Fixed.